components/event-feed-service/pkg/server/server.go

```diff
@@ -134,20 +143,20 @@ func (eventFeedServer *EventFeedServer) GetFeedTimeline(ctx context.Context,
 
 func (eventFeedServer *EventFeedServer) HandleEvent(ctx context.Context,
 	request *automate_event.EventMsg) (*automate_event.EventResponse, error) {
-	log.WithFields(log.Fields{
+	logctx := log.WithFields(log.Fields{
 		"request": request.String(),
-		"func":    "HandleEvent",
-	}).Debug("rpc call")
```

Comment:
While I agree with this removal, I vaguely remember superteam being appreciative of these log lines.